.env.template

```diff
@@ -1,6 +1,6 @@
-# Flask server configuration.
-FLASK_APP=todo_app/app
-FLASK_ENV=development
-
-# Change the following values for local development.
-SECRET_KEY=secret-key
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+# Change the following values for local development.
+SECRET_KEY=secret-key

```

Comment:
You could add your new env vars to this file (with placeholder values, not real ones) in order to make it easier for people to get started with the project.